src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Color {
+    Red,
+    Green,
+    Yellow,
+    White
```

Comment:
Enum constants should follow the naming conventions of constants and be in uppercase letters. For example, 'RED' instead of 'Red'.

Reply:
Fixed this comment